iceberg-handler/pom.xml

```diff
@@ -0,0 +1,252 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>org.apache.hive</groupId>
+        <artifactId>hive</artifactId>
+        <version>4.0.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>iceberg-handler</artifactId>
```

Comment:
Looks like the other handlers have this name pattern: hive-*-handler. Any reason the 'hive' part is omitted here?

Reply:
Good question. I've seen two patterns: 

- druid, hbase and kudu had the hive- prefix 
- kafka-handler did not have the prefix

I went with the second one, but I'm fine with adding the prefix. I guess it's the majority :)